### DISCUSSION TITLE
Lian Li SL v2 Fan control

### DISCUSSION THREAD

**Discussion by Tech127x** [Rem0o/FanControl.Releases#2186] [Q&A]
I have 4 banks of 3 fans.  Top rad push/pull, bottom rad push/pull.  At first, I was using 2 Lian Li controllers:

Bank 1 to controller 1, port 3
Bank 2 to controller 1, port 4
Bank 3 to controller 2, port 3
Bank 4 to controller 2, port 4

I've been using the plug-in from EightB1ts.  It sort of worked.  I could control all fans on controller 1 through port 3 and all fans on controller 2 through port 3.

I did some serious wire management yesterday, removed one of the Lian Li controllers, and now have all 12 fans connected to one controller.  Three per port on the controller so for clarity:

Bank 1 to controller 1, port 1
Bank 2 to controller 1, port 2
Bank 3 to controller 1, port 3
Bank 4 to controller 1, port 4

Now the plug-in doesn't work at all.  No matter what combination of controls/sensors I try in FanControl=nothing.

I removed the plug-in, and now can control all 12 fans at once through control/sensor 4 from the mobo.  That's all I can do.  This isn't exactly optimal for case flow management.  Obviously the top rad 6 group and bottom rad 6 group should all be the same speed, but it really seems like at bare minimum the top group and bottom group should be adjusted independantly.

Let me be clear: I am in no way going after Rem0o here.  I think FanControl is amazing!!!

Is there a plug-in from someone other than EightB1ts available?  Some way to fix his so it actually works?

Thx y'all!

**Comment by Rem0o** [comment]
I'm not all too familiar with Lian-li stuff, but isnt there some kind of settings in L-Connect to allow external control of the fans, some kind of toggle switch?

**Comment by Tech127x** [comment]
Thanks for the reply!  Unfortunately, not that I can find.  I did install L-Connect3 to check, but don't see anything anywhere in there so I uninstalled it.  Closed SignalRGB background too even though fan control is disabled in there.  I know what can happen when control softwares compete and didn't want to add that to the camel's back.

**Reply by Tech127x** [reply]
Update: I reinstalled LConnect3 again with a newer version Lian Li posted on FB of all places.  Yes, it was legit lol.  Anyway, now I can't control the fans with their software either.   After a little digging, I think it may be because I have the controller connected to an internal USB hub.  I don't know if the hub is the issue, but it seems I have to have that LL controller connected to USB header 2 not 1 or it won't work and I'm pretty sure that hub is connected to header 1.  This is just some google diagnosis but a lead.  Thanks for pointing me that direction!  I will try rerouting the USB (which isn't even close to easy) later today.  For now it can all run on fan mobo header 3 through FC until I'm motivated to tear into it again after working on cable management for two straight days.
